observer/observer.go

```diff
@@ -133,3 +94,15 @@ func getDirection(tx blockatlas.Tx, address string) blockatlas.Direction {
 	}
 	return blockatlas.DirectionOutgoing
 }
+
+func inferUtxoValue(tx *blockatlas.Tx, address string, coinIndex uint) {
+	if len(tx.Inputs) > 0 && len(tx.Outputs) > 0 {
```

Comment:
maybe return an error here?

Reply:
no, because if is not an utxo tx, the method don't going do to anything

Reply:
Maybe add property to transaction as type, and only call this function if it’s utxo transaction type?